docs/index.html

```diff
@@ -80,120 +80,122 @@ <h2>What is the purpose?</h2>
         label: "DDD",
         website: "",
         active: true,
-        moved: 0
+        moved: 0,
+        description:`
+        <p>Domain Driven Design (DDD) is an approach to software development that connects the implementation with an evolving model.</p>
```

Comment:
It's usually spelled with a hyphen ([Wikipedia](https://www.wikiwand.com/en/Domain-driven_design), [Literature](https://www.amazon.de/Domain-Driven-Design-Tackling-Complexity-Software/dp/0321125215)), so I'd do the same here.

```suggestion
        <p>Domain-Driven Design (DDD) is an approach to software development that connects the implementation with an evolving model.</p>
```